jaulp.wicket.base/src/main/java/de/alpharogroup/wicket/base/util/WicketComponentExtensions.java

```diff
@@ -321,6 +321,8 @@ public static void setSecurityHeaders(final WebResponse response)
 		response.setHeader("Expires", "-1");
 		// Set-Cookie
 		response.setHeader("Set-Cookie", "secure;httponly;");
+		// Set header value for 'Content-Security-Policy'

```

Comment:
Please see what exactly this header makes:
http://www.html5rocks.com/en/tutorials/security/content-security-policy/